src/PlaylistContainer/PlaylistContainer.tsx

```diff
@@ -1,41 +1,27 @@
 import React from 'react';
-import SavedPlaylist from '../SavedPlaylists/SavedPlaylist';
+
 import GeneratedPlaylist from '../GeneratedPlaylists/GeneratedPlaylist';
-// import CustomPlaylist from '../CustomPlaylists/CustomPlaylist';
 
 import { Playlist } from '../utils';
 
 interface IProps {
-	playlistType: string,
 	selectedGenre: string,
 	playlists: Playlist[] | [],
 	toggleSaved: (playlist: Playlist) => void
 }
 
-enum PlaylistTypes {
-	SavedPlaylist = 'saved-playlist',
-	GeneratedPlaylist = 'generated-playlist',
-	// CustomPlaylist = 'custom-playlist',
-}
-
-const displayPlaylist = (playlistType: string, playlists: any, selectedGenre: string, toggleSavedPlaylist: (playlist: Playlist) => void) => {
-	if (playlistType === PlaylistTypes.SavedPlaylist) {
-		return <SavedPlaylist />;
-	}
-	if (playlistType === PlaylistTypes.GeneratedPlaylist && playlists.length) {
+const displayPlaylist = (playlists: any, selectedGenre: string, toggleSavedPlaylist: (playlist: Playlist) => void) => {
+	if (playlists.length) {
```

Comment:
This was a good refactor, cleaning out the code and logic that is no longer necessary for conditionally rendering was the right call.